source/reducers/storageReducer.ts

```diff
@@ -13,7 +13,7 @@ export default (state: RootState, action: Action): RootState => {
 		case 'LOAD_SIMULATION_LIST':
```

Comment:
Est-ce que ce reducer reste appelé quelque part ? Ou est-ce une relique d'une itération précédente ?